lib/methods/interval.js

```diff
@@ -10,6 +10,9 @@ exports.on = {
 exports.define = {
   setInterval (id, fn, time) {
```

Comment:
this is unrelated to this pull request, but i think it would be nice if we kept a compatible signature with the vanilla setInterval method. 
i mean:
```javascript
setInterval(fn, time, id){
  id = id || 'val'
  // would get rid of lines 13 / 17
}
```